ada/ast.py

```diff
@@ -2908,6 +2944,31 @@ def previous_part_env_name():
             Self.top_level_env_name.to_symbol
         )
 
+    @langkit_property(return_type=T.env_assoc)
+    def previous_part_link_env_assoc():
+        """
+        Return the env association that describes where to add a `__nextpart`
+        entry for this body, if it corresponds to a non-overloadable entity
+        (i.e not a subprogram).
```

Comment:
i.e.* (missing dot)

Reply:
Btw, this also applies to entries, right? Do we consider them as subprograms?

Reply:
Thanks for raising this, I will add a small comment to say this isn't used by entries. In short: entry navigation is not handled the same way because they can never appear as a top level entity and are always contained in a declaration (task or protected object), which itself relies on that mechanism.

Reply:
Done!

Reply:
Ok thanks!